.github/workflows/template-build-test.yml

```diff
@@ -86,12 +81,11 @@ jobs:
         run: dotnet build -c Release --no-restore
 
       - name: Test
-        run: dotnet test -c Release --no-build --no-restore --blame-hang --blame-hang-timeout 60s /property:GenerateFullPaths=true /consoleloggerparameters:NoSummary /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura /p:ExcludeByAttribute='ExcludeFromCodeCoverageAttribute' /p:ExcludeByAttribute='CompilerGeneratedAttribute' /p:ExcludeByFile='**/*.g.cs' /p:Include='[MudBlazor]*' /p:SkipAutoProps=true /p:CoverletOutput='./TestResults/'
```

Comment:
Just wondering why you removed the include.  Doesnt this now show code coverage outside the library

Reply:
Yeah I think this is required for the Source Generator and the Analyser. Both have tests but the coverage was not included.

Reply:
How is the coverage excluded from the other projects are they annotated?